Services/Catalogue/Mappers/ConfigurableProductsData.php

```diff
@@ -266,6 +277,30 @@ private function getCategoryArray($category)
             $parentUrlKey = '';
         }
 
+        if ($this->allStoreBaseUrls === null) {
+            $this->allStoreBaseUrls = $this->storeManager->getAllStoreBaseUrls();
+        }
+
+        $storeUrls = $this->allStoreBaseUrls;
```

Comment:
$storeUrls = $this->allStoreBaseUrls;
Simply us $this->allStoreBaseUrls

Not required